doc/nrf/libraries/networking/nrf_cloud.rst

```diff
@@ -145,6 +145,26 @@ When the device receives the :c:enumerator:`NRF_CLOUD_EVT_FOTA_DONE` event, the
 The message payload of the :c:enumerator:`NRF_CLOUD_EVT_FOTA_DONE` event contains the :c:enum:`nrf_cloud_fota_type` value.
 If the value equals :c:enumerator:`NRF_CLOUD_FOTA_MODEM`, the application can optionally avoid a reboot by performing reinitialization of the modem and calling the :c:func:`nrf_cloud_modem_fota_completed` function.
 
+Building FOTA images
```

Comment:
This is rather generic stuff (except for the Cloud part of it) and I guess it could be better placed in, for example in the Getting started/Building and programming an application section (https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/gs_programming.html). Or then the Application development/Build and configuration system (https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/app_build_system.html). The generic part could then be included here from the place where this was added. Now, the changelog entry is placed under the Application development section, although this is documented only in this nRF Cloud library doc. If you decide to have this section only in this library doc, please move the changelog entry to the proper place under Libraries/Libraries for networking.

Reply:
My colleague also commented offline: "Rather than misplaced, it's hard to understand if nRF Cloud FOTA is using any of the FOTA libraries from the NCS, like FOTA download, or the Bootloader/DFU processes."

Reply:
@peknis I do not think the "generic stuff" belongs under Libraries for networking.  The changes I made in the PR were not specific to nRF Cloud, but were certainly relevant to it.  The changes are in general application build system locations in the source tree (/cmake, /modules/mcuboot, /scripts/bootloader), so the changelog entry should be where I put it, in my opinion.

I'm fine with moving the first three paragraphs to the second link you mention and simply reference that here.

Regarding what dependencies the nRF Cloud FOTA library has on other FOTA libraries, I agree it should be documented.  The file subsys/net/lib/nrf_cloud/Kconfig.nrf_cloud_fota forces on a number of libraries or features on.